### ISSUE TITLE
Error while testing build of 0.1.1

### ISSUE THREAD

**Issue by mucaho** [podhmo/python-semver#4]
I get an `AssertionError` when trying to build on linux with node-semver@0.1.1.
```shell
running build_ext
============================= test session starts ==============================
platform linux -- Python 3.5.3, pytest-3.0.7, py-1.4.33, pluggy-0.4.0
rootdir: /tmp/nix-build-python3.5-node-semver-0.1.1.drv-0/node-semver-0.1.1, inifile: setup.cfg
collected 993 items

semver/tests/test_comparators.py ..................................................................
semver/tests/test_comparison.py ..................................................................................................................................................................................................................................................................................................
semver/tests/test_equality.py ..................................................................................................................................................................................................................................................................................................................................................................................
semver/tests/test_increment_version.py ...................................
semver/tests/test_invalid_version_number.py .....
semver/tests/test_max_satisfying.py ....
semver/tests/test_negative_range.py ....................................................F.
semver/tests/test_range.py .......................................................................................
semver/tests/test_recovery.py ...
semver/tests/test_strict_vs_loose_version_number.py .......
semver/tests/test_valid_range.py ........................................................................

=================================== FAILURES ===================================
______ test_it[git+https://user:password0123@github.com/foo-123.0.0-True] ______

range_ = 'git+https://user:password0123@github.com/foo', version = '123.0.0'
loose = True

    @pytest.mark.parametrize("range_, version, loose", cands)
    def test_it(range_, version, loose):
        from semver import satisfies
>       assert (not satisfies(version, range_, loose)) is True
E       AssertionError: assert not True is True
E        +  where True = <function satisfies at 0x7ffff1746d90>('123.0.0', 'git+https://user:password0123@github.com/foo', True)

semver/tests/test_negative_range.py:72: AssertionError
============================ pytest-warning summary ============================
WC1 None [pytest] section in setup.cfg files is deprecated, use [tool:pytest] instead.
=========== 1 failed, 992 passed, 1 pytest-warnings in 1.89 seconds ============
```

**Comment by podhmo** [comment]
ok. changed, next version hasn't fail test.

**Comment by podhmo** [comment]
fixed in 0.2.0
